test/data/bad/bad-attribution-not-string.json

```diff
@@ -0,0 +1,7 @@
+{
+	"tilejson": "2.2.0",
+	"attribution": 12,
+	"tiles": [
+		"http://localhost:8888/admin/1.0.0/world-light,broadband/{z}/{x}/{y}.png"
+	]
+}
```

Comment:
Our team generally prefers spaces to tabs. Consider using spaces instead of tabs in this file and the others.